api/src/test/java/org/mandulis/mts/controller/DatabaseHelper.java

```diff
@@ -0,0 +1,59 @@
+package org.mandulis.mts.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.mandulis.mts.entity.Category;
+import org.mandulis.mts.entity.Ticket;
+import org.mandulis.mts.entity.User;
+import org.mandulis.mts.repository.CategoryRepository;
+import org.mandulis.mts.repository.TicketRepository;
+import org.mandulis.mts.repository.UserRepository;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+class DatabaseHelper {
```

Comment:
this class shouldn't be in a controller package. perhaps a util package would be more suitable